CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ the code was deployed.
 
 ## [Unreleased]
 
+### Added
+
+- Added 'View Client' button to Internal Dashboard-Location page (CU-8678vn1r5)
```

Comment:
Could you please add a period to the end of this line, just to be consistent with the rest of the file? (Although I see some other inconsistencies in there too. Feel free to change them to add periods too, if you'd like).

Reply:
Done!

Reply:
Thanks!